docs/pages/indexer.mdx

```diff
@@ -0,0 +1,133 @@
+# Indexer
+
+Indexer is an off-chain indexer for on-chain applications build with MUD.
+
+## Why an off-chain indexer?
+
+Reads with on-chain apps can be tricky.
+What does it mean to be able to query the Ethereum network?
+Technically, given a node with a fully synced state, we can explore just about everything using the EVM, but the “exploring” would be looking at raw storage slots for accounts corresponding to smart contracts.
+A way around this exists by providing view functions on smart contracts: these effectively are just wrappers around raw storage and expose a more friendly API.
+Instead of having to figure out where the balances for an account are stored in the storage tree, we now can call a function that does the lookup via Solidity via an RPC endpoint.
+
+The issue with view functions is that complexity in having to generate and call these to get the “full picture” of the state from the chain explodes pretty quickly.
+For a web app client wanting to present the user with an up-to-date view of the on-chain state, the task of calling the different view functions becomes unnecessarily hard very fast.
+This can also quickly accelerate the need to run a set of dedicated nodes just to be able to service the needed number of RPC requests.
+
+## Installation
+
+These are the steps to install and start Indexer.
```

Comment:
```suggestion
These are the steps to install and start the MUD indexer.
```

We don't really have a "brand name" for the indexer yet, would just call it "the indexer" or "the MUD indexer" for now

Reply:
OK. Thank you for changing one instance, I'll change the rest.